extension-config.json

```diff
@@ -1,7 +1,64 @@
 {
   "version": "1.0.0",
   "id": "@shopgate-project/smile-io",
-  "components": [],
-  "configuration": {},
+  "components": [
+    {
+      "id": "subscriptions",
+      "path": "frontend/subscriptions/index.js",
+      "type": "subscribers"
+    },
+    {
+      "id": "smileDataDigest",
+      "path": "frontend/reducers/simileDataDigest.js",
+      "type": "reducers"
+    },
+    {
+      "id": "SmileContainer",
+      "type": "portals",
+      "path": "frontend/portals/SmileContainer",
+      "target": "view.content.after"
+    }
+  ],
+  "configuration": {
+    "smileApiSecret": {
+      "type": "admin",
+      "destination": "backend",
+      "default": "",
+      "params": {
+        "label": "Smile api secret",
+        "type": "text",
+        "required": true
+      }
+    },
+    "smileChannelApiKey": {
+      "type": "admin",
+      "destination": "both",
+      "default": "",
+      "params": {
+        "label": "Smile channel API key",
+        "type": "text",
+        "required": true
+      }
+    },
+    "smileJavascriptSdkSrc": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "https://cdn.sweettooth.io/assets/storefront.js",
+      "params": {
+        "label": "Smile Javascript SDK script src",
+        "type": "text",
+        "required": true
+      }
+    },
+    "isLegacyCheckout": {
```

Comment:
shouldn't we use the real webcheckout shop config here to auto detect if it is a web or legacy checkout?

Reply:
added this